_maps/RandomRuins/IceRuins/icemoon_underground_abandoned_plasma_facility.dmm

```diff
@@ -662,7 +662,7 @@
 /area/ruin/plasma_facility/commons)
 "kA" = (
 /obj/structure/closet/crate{
-	icon_state = "crateopen"
+	opened = 1
 	},
```

Comment:
But if we are creating a separate subtype called opened won't mappers already know it's meant to be opened and the icon state will be redundant

Reply:
it's useful for things to be obvious visually, at least so long as this stuff is handled by icon state and not overlay

Reply:
I'm a bit confused if i create a subtype called
```
/obj/structure/closet/open
   opened = 1
```

Then from the name mappers can know this closet is in open state but in the map files if i do 
/obj/structure/closet/open{
   icon_state = "crateopen"
}

This is even more redundant because icon_state will be changed anyway by update_appearance() based on `opened`. Well it doesn't harm anyone and for visual clarity sake sure i'll do that

Reply:
I'll put the preopen then yup

Reply:
Excellent!